cmp/grid/Grid.scss

```diff
@@ -40,6 +40,11 @@
     cursor: pointer;
     width: 100%;
 
+    &--hidden .ag-header {
+      border-bottom-width: 0;
+      min-height: 0 !important;
+    }
```

Comment:
I'm assuming the `min-height` rule is doing something, since you've added it in here, but was a bit surprised as we set the header height to zero in code using the `headerHeight` getting in `AgGrid.ts`. What were you seeing without this rule?  Was there a 1px gap or something like that?

Reply:
![image](https://github.com/xh/hoist-react/assets/71867199/17391535-057f-4642-bed7-202f2ca62762)
Right Without the min-height there is a 1px gap